meta-openbmc-machines/meta-openpower/conf/machine/include/openpower.inc

```diff
@@ -4,6 +4,7 @@ OBMC_MACHINE_FEATURES += "\
         obmc-phosphor-sensor-mgmt \
         obmc-phosphor-flash-mgmt \
         obmc-phosphor-event-mgmt \
+        obmc-phosphor-user-mgmt \
```

Comment:
If this is going to be a feature (something that can be turned on/off), it should be a distro feature since managing users doesn't have anything to do with what kind of system it is.  For now the best thing to do is just not make this a feature at all and just put it in every image ( so just remove this line ).